src/Expander/Error.hs

```diff
@@ -204,11 +203,11 @@ instance Pretty VarInfo ExpansionErr where
     vsep [ pp env stx <> text ":"
          , group $ vsep [ group $ hang 2 $
                           vsep [ text "Used in a position expecting"
-                               , pp env (unExpected shouldBe)
+                               , pp env (unTenon shouldBe)
```

Comment:
Also backwards: this part of the error message is talking about what the position expects, so that's the mortise, while the other part is talking about what we're putting in that position, so that's the tenon.